Example/Showcase/Classes/DomainLayer/SocketFactory/SocketFactory.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Starscream
+import WalletConnectRelay
+
+struct SocketFactory: WebSocketFactory {
```

Comment:
You can use DefaultSocketFactory from Shared folder. Just include it in WalletApp target